yarn.lock

```diff
@@ -219,6 +219,46 @@
     prettier "^1.19.1"
     yargs "^15.1.0"
 
+"@aws-amplify/graphql-transformer-core@^0.17.12":
```

Comment:
that is unexpected to happen.
when you hoist cli locally it changes lock file. but I think that change should not be commited.

Reply:
+1, looks like this was committed after some test-related execution, we shouldn't have these in the package lock.

Reply:
sorry just saw this, let me update to remove